forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabMergeRequest.java

```diff
@@ -77,7 +77,13 @@ public String id() {
 
     @Override
     public HostUser author() {
-        return repository.forge().user(json.get("author").get("username").asString()).get();
+        var username = json.get("author").get("username").asString();
+        var author = repository.forge().user(username);
+        if (author.isPresent()) {
+            return author.get();
+        } else {
+            throw new IllegalArgumentException("Unknown username: " + username);
```

Comment:
I don't think IllegalArgumentException is correct here. It's meant to be thrown when a method receives a bad argument, but this method doesn't take arguments. We don't really have a reason to define a new exception type either, so a plain `RuntimeException` is probably better, at least for now.

I'm usually for short and concise messages, but in this case I think we can add some more details. Something like `"Author of merge request unknown: " + username`

Reply:
Ok, I also thought about the type of exception yesterday. I was thinking that the username is an argument of `user(username)`, but throw exception in method `user` is a little complex. 

I will change it to RuntimeException and make the error message have more details.